### DISCUSSION TITLE
Not receiving password reset emails

### DISCUSSION THREAD

**Discussion by leesus** [supabase/supabase#2459] [Questions]
I have 2 Supabase projects for my app, one for production and one for development. When I request a password reset email via my app (API calls `supabase.auth.api.resetPasswordForEmail(email)`) or the `Send password recovery email` in the Supabase dashboard, I don't receive the emails from either - both methods add an entry to the dashboards audit logs however. This issue persists across both projects. Nothing is getting flagged as spam, and sending an invite user email from the dashboard does get through after a few minutes, so it seems it is just password reset emails getting stopped.

Locally I'm using the docker image, and password reset emails are generated there correctly, so I don't believe it's an issue with my app - especially as the option to send a reset email in the UI doesn't work either. I do have the site urls set correctly, I can't see any other settings that would be related.

**Comment by silentworks** [comment]
This is a weird one, can you go to your **Authentication > Templates** in the Supabase dashboard and see if the **Reset password** template is filled in, please?

Also are you using the normal Supabase emailing service or have you setup your own custom SMTP?

**Reply by leesus** [reply]
The email templates are set to the defaults, and I'm using the Supabase email service.

What is stranger, is that other users do receive the password reset emails, so it appears to be an issue with just one email address.

**Reply by silentworks** [reply]
Which email service is this email address with? are their any out of the ordinary characters in the email address? what happen if you delete that account and add it again?

**Reply by leesus** [reply]
Sorry, forgot to update this.

The email address is an alias in G Suite. The default G Suite address received Supabase emails fine, but the alias didn't (I checked inbound mail logs in Google Admin and they weren't being received). No strange characters, and the alias can receive mail from other services and email addresses.

Deleting the account (and changing all the many references to the user id) would have been a bit of a pain, so I just used the reset token in the auth users table to construct my own reset link, which enabled me to change the password.

A bit odd, but there we go.
